README.md

```diff
@@ -1479,7 +1482,7 @@ Other Style Guides
 ## Iterators and Generators
 
   <a name="iterators--nope"></a><a name="11.1"></a>
-  - [11.1](#iterators--nope) Don’t use iterators. Prefer JavaScript’s higher-order functions instead of loops like `for-in` or `for-of`. eslint: [`no-iterator`](https://eslint.org/docs/rules/no-iterator) [`no-restricted-syntax`](https://eslint.org/docs/rules/no-restricted-syntax)
+  - [11.1](#iterators--nope) Don’t use iterators. Prefer JavaScript’s higher-order functions instead of loops like `for`, `for-in` or `for-of`. eslint: [`no-iterator`](https://eslint.org/docs/rules/no-iterator) [`no-restricted-syntax`](https://eslint.org/docs/rules/no-restricted-syntax)
```

Comment:
Yes, I agree. But maybe it does make sense to mention `for` loop somewhere here in this section as well (probably in the Note or separate bullet point)?
Because it is really very often used to iterate over Arrays and other iterables instead of array methods.

Reply:
not in a section about iterators tho.

if we mentioned it elsewhere, it'd be to avoid using loops at all, and that's a bit more controversial.